Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2008-11-21
Initial Package Version: 1.2.2
Upstream Status: unknown
Origin: fedora
Description: Allows it to compile with gcc-4.3
 I also had to pass '--without-xdao --without-lame' which is how fedora build it,
as there are similar issues somewhere else in the package.

diff -up cdrdao-1.2.2/trackdb/FormatConverter.cc.gcc43 cdrdao-1.2.2/trackdb/FormatConverter.cc
--- cdrdao-1.2.2/trackdb/FormatConverter.cc.gcc43	2008-02-18 13:17:53.000000000 +0100
+++ cdrdao-1.2.2/trackdb/FormatConverter.cc	2008-02-18 13:18:11.000000000 +0100
@@ -18,6 +18,7 @@
  */
 
 #include <stdlib.h>
+#include <string.h>
 #ifdef HAVE_AO
 #include <ao/ao.h>
 #endif
diff -up cdrdao-1.2.2/trackdb/TempFileManager.cc.gcc43 cdrdao-1.2.2/trackdb/TempFileManager.cc
--- cdrdao-1.2.2/trackdb/TempFileManager.cc.gcc43	2005-04-22 04:01:46.000000000 +0200
+++ cdrdao-1.2.2/trackdb/TempFileManager.cc	2008-02-18 13:14:14.000000000 +0100
@@ -24,6 +24,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <string.h>
 
 #define DEFAULT_TEMP_PATH "/tmp/"
 
